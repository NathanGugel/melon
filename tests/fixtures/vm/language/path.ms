func testPath |pathIn| => {
    let funcs = ["dirname", "basename"];

    for (let f in funcs) {
        io.print(path[f](pathIn));
    }
}

testPath("");
testPath("test");
testPath("test.ext");
testPath("/test.ext");
testPath("/");
testPath("..");
testPath("very/long/relative/path/to/file.ext");
testPath(this.path);
