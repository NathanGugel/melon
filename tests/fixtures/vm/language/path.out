.
.
.
test
.
test.ext
/
test.ext
/
/
.
..
very/long/relative/path/to
file.ext
fixtures/vm/language
path
